src/irr/asset/CGLSLVertexUtilsBuiltinIncludeLoader.h

```diff
@@ -19,9 +19,28 @@ class CGLSLBRDFBuiltinIncludeLoader : public irr::asset::IBuiltinIncludeLoader
 R"(#ifndef _IRR_VERTEX_UTILS_INCLUDED_
 #define _IRR_VERTEX_UTILS_INCLUDED_
 
-vec3 pseudoMul4x4with3x1(in mat4 m, in vec3 v)
+struct irr_glsl_SBasicViewParameters
 {
-    return (m[0]*v.x+m[1]*v.y+m[2]*v.z+m[3]).xyz;
+    mat4 MVP;
+    mat4x3 MV;
+    mat4x3 NormalMatAndEyePos;
+};
+mat3 irr_glsl_SBasicViewParameters_GetNormalMat(in irr_glsl_SBasicViewParameters _params)
+{
+    return mat3(_params.NormalMatAndEyePos);
+}
+vec3 irr_glsl_SBasicViewParameters_GetEyePos(in irr_glsl_SBasicViewParameters _params)
+{
+    return _params.NormalMatAndEyePos[3];
+}
```

Comment:
I don't think that struct and its methods belong in "vertex" utilities per-se (same could be used in fragment shaders, etc).

Reply:
yea, i didnt know where to put them. Another builtin loader?

Reply:
or just different builtin include "file"

Reply:
> or just different builtin include "file"?

yeah, I guess something like `irr/builtin/glsl/`+`common/parameters/SBasicViewParameters`